packages/quantic/force-app/main/default/lwc/caseAssistUtils/caseAssistUtils.js

```diff
@@ -0,0 +1,24 @@
+// @ts-ignore
+import getHeadlessConfiguration from '@salesforce/apex/HeadlessController.getHeadlessConfiguration';
+
+export async function getCaseAssistId(configName) {
```

Comment:
I'm not sure this should be in main/default as I don't think this should be packaged. This util was meant for testing purposes so we could manage configs without needing to make PRs if they change.
Why was it moved here?

Reply:
That was a mistake I will move this to be in the right location👌

Reply:
All right thanks